src/webgl/shaders/lighting.glsl

```diff
@@ -114,7 +114,7 @@ vec3 calculateImageDiffuse( vec3 vNormal, vec3 vViewPosition ){
   vec3 worldCameraPosition =  vec3(0.0, 0.0, 0.0);  // hardcoded world camera position
   vec3 worldNormal = normalize(vNormal);
   vec2 newTexCoor = mapTextureToNormal( worldNormal );
-  vec4 texture = TEXTURE( environmentMapDiffused, newTexCoor );
+  vec4 texture = TEXTURE_CUBE( environmentMapDiffusedCubemap, newTexCoor );
```

Comment:
I wonder if the naming of `texture` is an issue here since in a WebGL 2 environment, `TEXTURE_CUBE` is also `#define`d as the builtin function `texture()`? Does the error go away if you rename this variable?